utils/lib/pb_type_xml.py

```diff
@@ -100,6 +100,19 @@ def add_switchblock_locations(xml):
     })
 
 
+def get_site_pin_wire(equivalent_site_pin, site_pins):
+    no_prefix_equivalent_site_pin_wire = equivalent_site_pin.wire.split('_'
```

Comment:
This needs a docstring

Reply:
Done, actually I noticed I had a way to complex logic to check the site pin names, I have simplified that alongside with the docstrings addition.